source/administration/config-settings.rst

```diff
@@ -348,27 +352,31 @@ Restrict the permission level required to rename and set the header or purpose f
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"RestrictPrivateChannelManagement": "all"`` with options ``all``, ``channel_admin``, ``team_admin``, and ``system_admin`` for above settings respectively.   |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 Enable private group deletion for
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Restrict the permission level required to delete private groups. Deleted groups can be recovered from the database using a `command line tool <https://docs.mattermost.com/administration/command-line-tools.html>`_. The last member of a private group has the ability to delete the group regardless of their permission level.
 
 **All channel members**: Allow all group members to delete private groups.
 
-**Team and System Admins**: Restrict deleting private groups to Team and System Admins that are members of the group.
+**Channel Admins, Team Admins, and System Admins**: Restrict deleting private groups to Channel Admins, Team Admins, and System Admins that are members of the group.
+
+**Team Admins and System Admins**: Restrict deleting private groups to Team Admins and System Admins that are members of the group.
 
 **System Admins**: Restrict deleting private groups to System Admins that are members of the group.
 
-+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"RestrictPrivateChannelDeletion": "all"`` with options ``all``, ``team_admin`` and ``system_admin`` for above settings respectively.   |
-+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"RestrictPrivateChannelDeletion": "all"`` with options ``all``, ``channel_admin``, ``team_admin``, and ``system_admin`` for above settings respectively.   |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 Allow which users to delete messages
```

Comment:
"Allow which users to delete messages" > "Allow which users can delete messages" ?